tools/busco/main.nf

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env nextflow
+
+// Specify DSL2
+nextflow.preview.dsl = 2
```

Comment:
This is old syntax `nextflow.enable.dsl=2` use this where you can (we still need to go through and change this)

Reply:
dont worry ive done this in a PR for all